pytorch_lightning/accelerators/accelerator.py

```diff
@@ -84,7 +99,7 @@ def pre_dispatch(self, trainer: "pl.Trainer") -> None:
         if self.training_type_plugin.setup_optimizers_in_pre_dispatch:
             self.setup_optimizers(trainer)
 
-        self.precision_plugin.pre_dispatch()
+        self.training_type_plugin.precision_plugin.pre_dispatch()
```

Comment:
Should we use strategy already instead ?

Reply:
Agree we should unify the term. we haven't make that change yet? How about address in a different PR?

Reply:
Yes, the renaming has to happen in a separate PR and it will touch many files. This can definitely not be done here.